xcube_hub/core/cate.py

```diff
@@ -63,10 +62,12 @@ def launch_cate(user_id: str) -> JsonObject:
         cate_image = util.maybe_raise_for_env("CATE_IMG")
         cate_tag = util.maybe_raise_for_env("CATE_TAG")
         cate_hash = os.getenv("CATE_HASH", default=None)
+        cate_debug = os.getenv("CATE_DEBUG", default='0')
+        cate_launch_grace = os.getenv("CATE_LAUNCH_GRACE_PERIOD", default=2)
 
         cate_command = util.maybe_raise_for_env("CATE_COMMAND",
                                                 "cate-webapi-start -b -p 4000 "
-                                                "-a 0.0.0.0 -s 3600 -r"
+                                                "-a 0.0.0.0 -s 86400 -r"
```

Comment:
Add `-v` flag, otherwise we cannot see any log messages of level `"DEBUG"`. But that is required in development mode.